core-services/monitoring.tf

```diff
@@ -0,0 +1,30 @@
+module "alarm_channel" {
+  count       = true ? 1 : 0
+  source      = "github.com/massdriver-cloud/terraform-modules//k8s/alarm-channel?ref=41e799c"
+  md_metadata = var.md_metadata
+  namespace   = kubernetes_namespace_v1.md-observability.metadata.0.name
+  release     = var.md_metadata.name_prefix
+
+  depends_on = [module.prometheus-observability]
+}
+
+module "application_alarms" {
+  count             = true ? 1 : 0
+  source            = "github.com/massdriver-cloud/terraform-modules//massdriver/k8s-application-alarms?ref=41e799c"
+  md_metadata       = var.md_metadata
+  pod_alarms        = true
+  deployment_alarms = true
+  daemonset_alarms  = true
+
+  depends_on = [module.prometheus-observability]
+}
+
+module "cluster_autoscaler_max_scale" {
```

Comment:
We don't install the cluster autoscaler in GKE. We use a built in one (kinda like AKS). We'll need to hook into the clouds alarm. I think this is what we want:
https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-autoscaler-visibility#noscaleup-event

Reply:
Ah that's right you did mention that. I'll set that up